lib/manageiq/providers/workflows/builtin_runner.rb

```diff
@@ -0,0 +1,41 @@
+module ManageIQ
+  module Providers
+    module Workflows
+      class BuiltinRunnner < Floe::Runner
+        SCHEME = "builtin"
+
+        def run_async!(resource, env = {}, secrets = {})
+          scheme_prefix = "#{SCHEME}://"
+          raise ArgumentError, "Invalid resource" unless resource&.start_with?(scheme_prefix)
+
+          method = resource.sub(scheme_prefix, "")
+
+          # TODO: prevent calling anything except the specifics methods, e.g. you shouldn't be able to call .to_s.
+          #       Maybe make BuiltinMethods a BasicObject?
+          BuiltinMethods.public_send(method)
+        end
+
+        def cleanup(runner_context)
+        end
+
+        def wait(timeout: nil, events: %i[create update delete])
+          # TODO: wait_for_task?
+        end
+
+        def status!(runner_context)
+        end
+
+        def running?(runner_context)
+        end
```

Comment:
If some of the builtin methods we are going to call will be async we can use this method here to control when we call  back into the workflow to step through the next state.

We could define a `_ready?` method for each builtin that will indicate when it is complete (can just be `true` for fire-and-forget methods).